bin/generators/__init__.py

```diff
@@ -0,0 +1,71 @@
+import itertools
+
+import lib
+
+
+def word_generator(length_min=7, length_max=15, perms=False):
+    """
+      Generate the words to be used for bruteforcing
+      > :param length_min: minimum length for the word
+      > :param length_max: max length for the word
+      > :param perms: permutations, True or False
+      > :return: a word
+
+      Example:
+      >>> word_generator()
+      aaaaaa
+      aaaaab
+      aaaaac
+      ...
+    """
+    if not perms:
+        chrs = 'abc'
```

Comment:
Only `'abc'`? This seems very limited for bruteforcing.

Then again, using even just `a-zA-Z` is almost impossible (time wise) considering even a small number like `length_max=6`.

```python
sum(1 for _ in itertools.product(string.ascii_letters, repeat=1))
52
sum(1 for _ in itertools.product(string.ascii_letters, repeat=2))
2704
sum(1 for _ in itertools.product(string.ascii_letters, repeat=3))
140608
sum(1 for _ in itertools.product(string.ascii_letters, repeat=4))
7311616
sum(1 for _ in itertools.product(string.ascii_letters, repeat=5))
380204032
```

> "...it is possible that it could take a lifetime to successfully crack your hash" - You

As you know, bruteforcing in Python seems like a bad idea (it's just a relatively slow language). Dictionary attacks? Maybe. I'm just not sure if this bruteforce code really adds anything to the library.